udata/templates/user/card.html

```diff
@@ -2,8 +2,8 @@
 {% set user_url = url_for('users.show', user=user) %}
 <div class="card">
     <a class="card-logo" href="{{ user_url }}">
-        <img src="{{ user|avatar_url(48) }}" alt="{{ user.fullname }}"
-            width="48" height="48">
+        <img src="{{ user|avatar_url(60) }}" alt="{{ user.fullname }}"
+            width="60" height="60">
```

Comment:
Shouldn't we deal with Retina (here and everywhere else)? i.e. request a 120px avatar.

Reply:
Yes, but I think it requires a PR on its own (maybe dealing with `srcset`)